src/state/apis/zapper/validators.ts

```diff
@@ -13,7 +13,27 @@ import {
 import { invert } from 'lodash'
 import type { Infer, Type } from 'myzod'
 import z from 'myzod'
-import { isNonEmpty, isUrl } from 'lib/utils'
+
+// Redeclared here to make these two utils work in scripts
```

Comment:
Ah yes, I've tried to solve this conundrum before and ended up doing the same thing 😞